### PR TITLE
Greatly simplifies Docker image.

### PR DESCRIPTION
**Pull request by MicahZoltu**
Inlines stuff where possible and gets rid of all of the complexity around the start script.

Also adds .editorconfig and applies it.  Open to discussion about specific choices.

### CODE REVIEW

File: start.sh [hunk 1]
```diff
@@ -1,65 +1,14 @@
...
-    else
-      setup_chain_template
-      echo "${ROOT}/chain-template mounted, but uninitialized. Copying chaindata template"
-      mv ${ROOT}/chain-template/* ${ROOT}/chain
-    fi
-  fi
-}
-
-start
+# geth is dumb and won't let us run it in the background, and nohup redirects to file when run in a script
```
Comment (epheph) [inline start.sh]: comment still necessary?

Comment (MicahZoltu) [reply start.sh]: Fixed.

File: start.sh [hunk 2]
```diff
@@ -1,65 +1,13 @@
...
-    else
-      setup_chain_template
-      echo "${ROOT}/chain-template mounted, but uninitialized. Copying chaindata template"
-      mv ${ROOT}/chain-template/* ${ROOT}/chain
-    fi
-  fi
-}
-
-start
+geth \
```
Comment (epheph) [inline start.sh]: Once of the things that no longer happens here is the trap'ing and forwarding of signals (which we've seen geth to be very sensitive to).

Just adding `exec` before geth should fix it

**Review by epheph — APPROVED** [review]
comment and entrypoint

File: start.sh (hunk 2 above)
Comment (MicahZoltu) [reply start.sh]: Fixed.

**Pull request merged**